python-backend/main.py

```diff
@@ -107,12 +107,16 @@ async def data_socket(websocket: WebSocket):
             logger.debug(f"last convergent / limit: {limit}")
             await websocket.send_json({"limit": "Infinity" if type(limit) is Infinity else str(limit)})
 
+            # @TODO replace wide_search=True with wide_search=[1]
             computed_values = db.identify(values=[str(limit)], wide_search=True)
+            json_computed_values = []
```

Comment:
Extremely minor nit: The name of this variable caused me to wonder about the json-ness of the values, when they appeared to be strings. I see that they are then converted to json when transmitted by the websocket.

Recommendation: change to `computed_values: list[str] = []` so people don't look for why json is important in this context.